www/js/components/device-storage.js

```diff
@@ -16,25 +16,19 @@ export class DeviceStorage extends Component {
     );
   }
 
+  // TODO: Refactor style logic into CSS
```

Comment:
I made a git issue, you can remove this TODO
https://github.com/bikelomatic-complexity/planning/issues/113